src/briefcase/integrations/android_sdk.py

```diff
@@ -282,13 +281,14 @@ def install(self):
                 cmdline_tools_zip_path,
                 extract_dir=self.cmdline_tools_path.parent
             )
-        except (shutil.ReadError, EOFError):
+        except (shutil.ReadError, EOFError) as exc:
```

Comment:
Really minor consistency thing here; we've used `as e` everywhere except this once case.